### PR TITLE
Navigation added

### PR DESCRIPTION
**Pull request by akash-1318**

### CODE REVIEW

File: components/css/componentStyle/Navigation.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    width: 1.5rem;
+    background-color: tomato;
+    top: -5px;
+    right: 0px;
+    display: flex;
+    justify-content: center;
+    color: white;
+    font-size: 1rem;
+    border-radius: 50%;
+}
```
Comment (irajbhinde) [inline components/css/componentStyle/Navigation.css:98]: add align-items:center here to make the icons look good

File: components/css/componentStyle/Navigation.css [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    align-items: center;
+}
+
+.badge__content{
+    position: absolute;
+    height: 1.5rem;
+    width: 1.5rem;
+    background-color: tomato;
+    top: -5px;
+    right: 0px;
```
Comment (irajbhinde) [inline components/css/componentStyle/Navigation.css:91]: nothing major but badge on heart looks a bit odd, adding some right value would look better

**Review by irajbhinde — COMMENTED** [review]
Looks good, just do the minor change mentioned in comments.

File: components/css/componentStyle/Navigation.css [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    display: flex;
+    align-items: center;
+}
+
+.badge__content{
+    position: absolute;
+    height: 1.5rem;
+    width: 1.5rem;
+    background-color: tomato;
+    top: -5px;
```
Comment (ark2002) [inline components/css/componentStyle/Navigation.css]: would suggest using a color code instead of color name

**Review by ark2002 — CHANGES_REQUESTED** [review]
just the changes suggested rest looks good

**Pull request merged**